Project.toml

```
@@ -73,7 +73,7 @@ CUDA = "5.8"
 CodeTracking = "1.0.5, 2"
 ConstructionBase = "1.5"
 Convex = "0.16"
-DataStructures = "0.18.15"
+DataStructures = "0.18.15, 0.19"
```

```suggestion
DataStructures = "=0.19"
```
Try to force v0.19 to see, which package is holding it back.